src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt

```diff
@@ -215,6 +215,7 @@ class CodyToolWindowContent(private val project: Project) : UpdatableChat {
   }
 
   private fun loadCommands() {
+    tryRestartingAgentIfNotRunning(project)
```

Comment:
I tried `:runIde` a few times - with a rate-limited account I saw the issue with commands not loading - this line seems to be a fix